CMakeLists.txt

```diff
@@ -216,25 +260,21 @@ set(SYCLCC_CONFIG_FILE "{
 
 file(WRITE ${SYCLCC_CONFIG_FILE_PATH} ${SYCLCC_CONFIG_FILE})
 
-if(WIN32)
-  set(HIPSYCL_INSTALL_CMAKE_DIR
-    "cmake" CACHE PATH "Install path for CMake config files")
-else()
-  set(HIPSYCL_INSTALL_CMAKE_DIR
-    "lib/cmake/${PROJECT_NAME}" CACHE PATH "Install path for CMake config files")
-endif()
-
-configure_file(
-  ${PROJECT_SOURCE_DIR}/include/hipSYCL/common/config.hpp.in
-  ${PROJECT_BINARY_DIR}/include/hipSYCL/common/config.hpp)
+set(HIPSYCL_INSTALL_CMAKE_DIR
+  "lib/cmake/${PROJECT_NAME}" CACHE PATH "Install path for CMake config files")
 
 install(DIRECTORY include/CL DESTINATION include/ FILES_MATCHING PATTERN "*.hpp")
 install(DIRECTORY include/SYCL DESTINATION include/ FILES_MATCHING PATTERN "*.hpp")
-install(DIRECTORY include/hipSYCL DESTINATION include/ FILES_MATCHING PATTERN "*.hpp")
+install(DIRECTORY include/hipSYCL DESTINATION include/ FILES_MATCHING PATTERN "*.hpp" PATTERN include/hipSYCL/compiler EXCLUDE)
```

Comment:
Why is it important to not install the compiler headers? They are not used when compiling with `syclcc`, but maybe at some point someone would like to build something on top of the hipSYCL infrastructure?

Reply:
it's probably not.. this change is actually some months old, so I don't remember, why I thought it was that important..